profiles/standard

```diff
@@ -729,3 +729,6 @@ zypp.gui.pkexec.run auth_admin:auth_admin:auth_admin_keep
 # a daemon that deals with system power settings (bsc#1189900)
 net.hadess.PowerProfiles.switch-profile no:no:yes
 net.hadess.PowerProfiles.hold-profile no:no:yes
+
+# kcron dbus helper (bsc#1193945)
+local.kcron.crontab.save no:auth_admin:yes
```

Comment:
With `yes` for the third field (i.e. for active users) you would allow regular local users to modify the system crontab without authentication. That means the local user more or less can get root without authentication.

I don't think that is what we want, not even for the "easy" profile. I suggest `no:no:auth_admin` for restrictive and `no:no:auth_admin_keep` for standard and easy. The latter means that in the easy and normal profiles the user can subsequently change the system crontab from the same kcron instance without having to re-enter the password all the time.

Reply:
Yes you're right, thanks for the explanation! It should be fixed now :)